README.md

```diff
@@ -1,13 +1,13 @@
 # Infinity Works Data Test - Starter Project
 
-### Prerequisites
-#### Java JDK 8
+## Prerequisites
```

Comment:
Pretty certain it doesn't require Java; I've checked the modules, cloned and run the project without. If memory serves, the data _used_ to be generated with Java, not Python.